packages/app/src/cli/api/graphql/business-platform-organizations/queries/fetch_dev_store_by_domain.graphql

```diff
@@ -0,0 +1,20 @@
+  query FetchDevStoreByDomain($domain: String) {
+    organization {
+      id
+      name
+      properties(search: $domain, offeringHandles: ["shop"]) {
```

Comment:
Should this contain `filters: {field: STORE_TYPE, operator: EQUALS, value: "app_development"}`?